README.md

```diff
@@ -172,7 +172,101 @@ when they expire.
 **Important**: ``credentials.state`` changes after a token swap. Be sure to persist
 the new state.
 
-
+### Using OAuth2 Credentials
+
+OAuth2 Credentials work similarly to the 3-step OAuth used by
+Partner Applications. OAuth2 tokens have a 30 minute expiry, 
+but can be swapped for a new token at any time. Procedure for authenticating
+credentials:
+
+ 1) Construct an `OAuth2Credentials` instance. The callback URI should
+    be served by your app in order to complete the authorization:
```

Comment:
The callback URI must also match the name provided when you registered the app.

Reply:
I've addressed this in the documentation update.